### DISCUSSION TITLE
TIN, EIN data - the examples show formatted, the schema only indicates 'string'

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#95] [Q&A] [answered]
The examples are showing a formatted field but the schema only reflects as 'string'---so is the expectation to always format the data in the file?

So are we to format or not format the TIN, EIN data being published in the files?  
If there is a provider that is using their SSN, then are we to format it as SSN rather than EIN format?

For EIN/TIN are we to publish as 000000000 or 00-0000000
If provider has SSN rather than TIN are we to publish as 000000000 or 000-00-0000

**Comment by rgouldkc** [comment]
I am also wondering if there is a requirement to format TIN, SSN and EIN data. Typically in our systems this data is not formatted with - and my preference would be not to format it in the files we generate.

**Comment by ghost** [comment]
We are following the schema definition of a string as there is not format requested in the description like date fields whose description call out an ISO 8601 format (e.g. YYYY-MM-DD).

I agree with @rgouldkc that it should not be required to be formatted.

**Comment by shaselton-usds** [comment, marked as answer]
There are no formatting requirements for those values currently.
